src/state/apis/swappers/helpers/getIsDonationAmountBelowMinimum.ts

```diff
@@ -0,0 +1,20 @@
+import type { Asset } from 'lib/asset-service'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { fromBaseUnit } from 'lib/math'
+import { getMinimumDonationUsdSellAmountByChainId } from 'lib/swapper/swappers/utils/getMinimumDonationUsdSellAmountByChainId'
+
+export const getIsDonationAmountBelowMinimum = (
+  sellAmountIncludingProtocolFeesCryptoBaseUnit: string,
+  sellAsset: Asset,
+  sellAssetUsdRate: string,
+) => {
+  const sellAmountBeforeFeesCryptoPrecision = fromBaseUnit(
+    sellAmountIncludingProtocolFeesCryptoBaseUnit,
+    sellAsset.precision,
+  )
```

Comment:
Something is off here - we go from an amount _including_ fees to an amount _before_ fees.
The preference would be to use the before fees amount, though we might not have it in context, and the including fees was likely what we were using before anyway.

Reply:
Actually, it looks like we were using the before fee amount before - can we update the var names and use that value in the two usages of this helper?

Reply:
updated to align vernacular 8b8cc85be